save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/TestSuiteRepository.kt

```diff
@@ -36,9 +35,8 @@ interface TestSuiteRepository : BaseEntityRepository<TestSuite> {
         "FunctionNaming",
         "FunctionName",
     )
-    fun findAllBySource_Organization_NameAndSource_NameAndVersion(
-        organizationName: String,
-        sourceName: String,
+    fun findAllBySource_IdAndVersion(
```

Comment:
Why do you start to use `_` in names?

Reply:
As for me, it reads like `find by source, id and version` and not like `find by sourceId and version`, and that is confusing
at least for me

Reply:
yes, probably it's redundant here.
Will remove it here.
But I think `_`  is useful when we take entity inside another entity